assets/scripts/vendor/elementor.js

```diff
@@ -0,0 +1,17 @@
+/* global jQuery */
+
+(function pageLoad($) {
```

Comment:
```suggestion
(function ($) {
```

Reply:
![Screen Shot 2021-06-21 at 18 28 04](https://user-images.githubusercontent.com/6313979/122796228-6d0b2c00-d2be-11eb-8272-935495ff7016.png)

Reply:
I named it different now than the main.js function though.